src/scenic/simulators/carla/simulator.py

```diff
@@ -82,22 +95,27 @@ def __init__(self, scene, client, map, timestep, render, record, verbosity=0):
 			print("blueprint: ", blueprint)
 
 			# Set up transform
-			loc = utils.scenicToCarlaLocation(obj.position, z=obj.elevation, world=self.world)
+			loc = utils.scenicToCarlaLocation(obj.position, world=self.world)
 			rot = utils.scenicToCarlaRotation(obj.heading)
 			transform = carla.Transform(loc, rot)
-			
+			transform.location.z += obj.elevation
```

Comment:
I would rather have this in `scenicToCarlaLocation` (with the elevation getting passed in as an optional argument as before), since the point of that function is to do coordinate conversion. See also my comment in `model.scenic` about the default elevation being `None` or not.

Reply:
Check elevation comment